charts/nodejs/templates/scale-pods-right-to-api.yaml

```diff
@@ -0,0 +1,23 @@
+{{ if contains "api" .Values.biomageCi.repo }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: can-scale-worker-pods
+rules:
+- apiGroups: ["apps"]
+  resources: ["deployments"]
+  verbs: ["update", "get"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: api-can-scale-worker-pods
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: can-scale-worker-pods
+subjects:
+- kind: ServiceAccount
+  name: deployment-runner
+  namespace: {{ .Release.Namespace}}
```

Comment:
Doesn't work without this @ivababukova

Reply:
No worries, this must be the way to do it then.